src/contract_cleaning.py

```diff
@@ -0,0 +1,19 @@
+def get_one_hot_encoded_contract(contract):
+    contract_as_list = contract.values.tolist()  # Returns a list of list [[]]
+    contract_cleaned = []
+
+    for item in contract_as_list:
+        contract_cleaned.append(item[0])
```

Comment:
in complex cases, i agree that it obscures what's going on, but this is a pretty clear case where instead of reading/parsing and checking 4 lines, i can visually grasp what's going on in 1.

if i recall correctly it's also slightly faster, but don't quote me on that